### DISCUSSION TITLE
Set a custom start page after login

### DISCUSSION THREAD

**Discussion by martijnboland** [directus/directus#4590] [Feature Requests]
It would be great if we could customize which page is loaded in the Directus UI after logging in. Currently, it redirects to the last visited page, but personally I'd like to always redirect our users to a custom dashboard page.

Possible options for a project setting:
- Last visited page (current default)
- Collections overview
- Custom (and then offer a possibility to enter a custom url relative from the base url such as '/dashboard')
- ...

**Comment by benhaynes** [comment]
Love it. I think we should add this at the Role level, since access to certain pages may be different. We have a dropdown component that (I believe) supports an input for "other". This seems like the ideal way to implement. The only thing to keep in mind is that if there is a "ref" link (query parameter that tells the user where to go after login), that should take priority.

**Reply by martijnboland** [reply]
Yes, setting this per role would be even better since we already have different dashboards per role :-)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
